File: solutions/python/resistor-color-duo/2/resistor_color_duo.py
BANDS = ["black", "brown", "red", "orange", "yellow", "green", "blue", "violet", "grey", "white"]

def value(colors: list) -> int:
    """Gives the value of the first two colors from the input list
    
    :param colors: The list of colors from which the first two will be translated into the code.
    :return: int - The value of the first two colors combined.
    """
    
    return BANDS.index(colors[0]) * 10 + BANDS.index(colors[1])

File: solutions/javascript/resistor-color-duo/1/resistor-color-duo.js
/**
* This function takes a string that represents a color and checks it against a list, giving back its index which is also the numerical value of that color in Transistor code
* 
* @param colors An array with two or more colors to be decoded
* @returns The numerical value associated with each of the first two colors put together as a code
*/
export const decodedValue = (colors) => {
  return Number(
    String(COLORS.indexOf(colors[0])) + 
    String(COLORS.indexOf(colors[1]))
  );
};

/**
* The List that holds all colors used for resistors, each associated with the index that is also the value it represents
*/
export const COLORS = [
  "black",
  "brown",
  "red",
  "orange",
  "yellow",
  "green",
  "blue",
  "violet",
  "grey",
  "white"
];
  